traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -1459,10 +1460,8 @@ func Update(w http.ResponseWriter, r *http.Request) {
 	}
 
 	original := originals[0]
-	if original.XMPPID == nil {
-		sysErr = errors.New("original server had no XMPPID")
-		api.HandleErr(w, r, tx, http.StatusInternalServerError, nil, sysErr)
-		return
+	if original.XMPPID == nil || *original.XMPPID == "" {
+		log.Warnln("original server had no XMPPID")
```

Comment:
we should include more context, including the server hostname, in this log message

Reply:
Including the hostname in the warning in 034ad01440.